Sofee.Core.Api.Tests.Acceptance/Brokers/SofeeApiBroker.cs

```diff
@@ -11,15 +11,13 @@ namespace Sofee.Core.Api.Tests.Acceptance.Brokers
 {
     public partial class SofeeApiBroker
     {
-        private readonly WebApplicationFactory<Startup> webApplicationFactory;
-        private readonly HttpClient httpClient;
         private readonly IRESTFulApiFactoryClient apiFactoryClient;
 
         public SofeeApiBroker()
         {
-            this.webApplicationFactory = new WebApplicationFactory<Startup>();
-            this.httpClient = this.webApplicationFactory.CreateClient();
-            this.apiFactoryClient = new RESTFulApiFactoryClient(this.httpClient);
+            WebApplicationFactory<Startup> webApplicationFactory = new WebApplicationFactory<Startup>();
```

Comment:
In this case you would want to use `var` since the right hand side is clear
Refer to our C# coding Standard here:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#01-declarations

Reply:
Aah! Got it.